_Sidebar.md

```diff
@@ -8,37 +8,36 @@
     * [[UX researchers|Conducting-research-with-students]]
     * [[Voice artists | Instructions-for-voice-artists]]
     * [[Designers and artists|Contributing-to-Oppia's-design]]
-  * [[How to file a bug report|How-to-file-a-bug-report]]
-  * **[[Google Summer of Code 2024|Google-Summer-of-Code-2024]]**
+  * [[How to report a bug|How-to-file-a-bug-report]]
+  * [[Google Summer of Code 2024|Google-Summer-of-Code-2024]]
   * **[[Google Season of Docs 2024|Season-of-Docs-2024]]**
 ---
 **Developing Oppia**
   * [[FAQs|Frequently-Asked-Questions]]
   * [[Installing Oppia|Installing-Oppia]]
   * Getting started with the codebase
-    * [['Getting started' guide|https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up]]
+    * **[['Getting started' guide|https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up]]**
     * [[Populating data on local server|Populating-data-on-local-server]]
     * Tutorials
       * [[Make a simple UI change|Making-a-simple-UI-change]]
+      * [[Tips for finding the right code to change|Find-the-right-code-to-change]]
+      * [[Tips for analyzing the codebase|Analyzing-the-codebase]]
     * [[How to access Oppia webpages|How-to-access-Oppia-webpages]]
     * Team onboarding guides
       * [[LaCE Team|LaCE-onboarding-guide]]
       * [[Contributor Dashboard Team|Contributor-dashboard]]
-  * [[Upgrading python version to 3.8.15|Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory]]
```

Comment:
I assume we're dropping this from the sidebar because it's no longer needed. If that's the case, should we also delete the page? We can always recover it from the commit history if we want to refer to it later on, but at this point everyone should have done the Python upgrade

Reply:
Done. Thanks!